jobs/designer.md

```diff
@@ -0,0 +1,64 @@
+# UI/UX Designer
+
+We’re looking for an experienced and proactive UI/UX Designer who is not afraid of independent work and interested in discovering and learning new design tools. The job implies growing into a Design Team Lead.
+
+**Salary:** 2k-3k USD gross per month.
+
+
+## What will you do:
+- Design user-friendly interface for clients’ web and mobile applications.
+- Build UX part of the apps on the basis of the existing design system.
+- Create the whole design system for the internal products by a demand using AI tools.
+- Help other teams with design-related tasks: design website pages, edit photos’ size and shape, etc.
+- Manage and coach the design team.
+
+
+## What qualities do we expect you to have:
+- High level of expertise in UX.
+- Independency and ability to work without teammate’s guidance. We’re starting to build our design team, so some time you’ll work as a single designer in the department.
+- Fluency in English (writing and speaking) for daily use.
+- Understanding of the principles of building and managing the team.
+- Proactivity: ability to see points for improvement and initiativeness in proposing changes.
+- Versatileness: you’ll work on the client’s project, but sometimes we can ask you to help the team with internal tasks.
+- Interest in working with AI tools.
+
+## Would be a plus:
+- Knowledge of UI Kit, React, HTML.
```

Comment:
Actually, I'd reduce the amount of text in the last abstract and rewrite the last sentence. Let's put this version to Notion and finish editing here, because it seems to be faster.

Reply:
Sure!